Description: Ignore newlines from guest list
 The list file expects all guest UUIDs on the same line as the URI
 which the guests run on. This does not happen when the list is
 echo'ed in quotes. When stripping the quotes, newlines get transformed
 into spaces.
 Based on a fix by Omar Siam <simar@gmx.net>
Forwarded: no
Bug-Ubuntu: http://bugs.launchpad.net/bugs/1591695
Author: Stefan Bader <stefan.bader@canonical.com>

Index: libvirt/tools/libvirt-guests.sh.in
===================================================================
--- libvirt.orig/tools/libvirt-guests.sh.in	2016-10-06 12:19:36.088316335 +0200
+++ libvirt/tools/libvirt-guests.sh.in	2016-10-06 12:24:28.244311831 +0200
@@ -520,7 +520,7 @@ stop() {
         fi
 
         if [ -n "$list" ]; then
-            echo "$uri" "$list" >>"$LISTFILE"
+            echo "$uri" $list >>"$LISTFILE"
         fi
     done
     set +f
